index.js

```diff
@@ -47,6 +47,7 @@ module.exports = function (osm, media, opts) {
   router.addRoute('GET /sync/destroy', api.syncDestroy.bind(api))
   router.addRoute('GET /sync/join', api.syncJoin.bind(api))
   router.addRoute('GET /sync/leave', api.syncLeave.bind(api))
+  router.addRoute('GET /sync/clear', api.syncClear.bind(api))
```

Comment:
What should this do? Could you add a README entry for this route?

Reply:
this clears the sync state, which answers your other question on the pr about when does the 'open connections' property get cleared